desktop/wasi-mtt/src/impl/poll.rs

```diff
@@ -1,8 +1,8 @@
+use crate::{r#impl::pledge::contains, Context, Error, ErrorKind, Promises};
```

Comment:
Comments at the top of the file are useful like in the rest of files

Reply:
Yeah the APIs are all documented in their respective `.wit` IDL files, but I added a comment to each now that refers a reader to those files.